stdlib/stdlib.fc

```diff
@@ -440,7 +440,7 @@ builder store_ref(builder b, cell c) asm(c b) "STREF";
 ;;; Stores `slice` [s] into `builder` [b]
 builder store_slice(builder b, slice s) asm "STSLICER";
 
-;;; Stores (serializes) an integer [x] in the range `0..2^120 − 1` into `builder` [b].
+;;; Stores (serializes) an integer [x] in the range `0..2^128 − 1` into `builder` [b].
```

Comment:
These things keep bubbling up :)

Reply:
oh, again, haha 😆
it's from the copy-pasting the latest stdlib from Core team projects

Reply:
fixed